conda/environments/bench_ann_cuda-125_arch-aarch64.yaml

```diff
@@ -0,0 +1,42 @@
+# This file is generated by `rapids-dependency-file-generator`.
+# To make changes, edit ../../dependencies.yaml and run `rapids-dependency-file-generator`.
+channels:
+- rapidsai
+- rapidsai-nightly
+- dask/label/dev
+- conda-forge
+- nvidia
+dependencies:
+- benchmark>=1.8.2
+- c-compiler
+- clang-tools=16.0.6
+- clang==16.0.6
+- cmake>=3.26.4,!=3.30.0
+- cuda-cudart-dev
+- cuda-nvcc
+- cuda-nvtx-dev
+- cuda-profiler-api
+- cuda-python>=12.0,<13.0a0
+- cuda-version=12.5
+- cxx-compiler
+- cython>=3.0.0
+- dlpack>=0.8,<1.0
+- gcc_linux-aarch64=11.*
+- glog>=0.6.0
+- h5py>=3.8.0
+- hnswlib=0.7.0
```

Comment:
cuvs' cpp cmake patch has the hnswlib version as 0.6.2 so there is a mismatch. @divyegala can weigh in on this.

Reply:
We'll need to match versions for now until we can upgrade and create a new patch.